pkg/accounting/accounting.go

```diff
@@ -219,3 +226,34 @@ func (a *Accounting) getPeerBalance(peer swarm.Address) (*PeerBalance, error) {
 func (pb *PeerBalance) freeBalance() int64 {
 	return pb.balance - int64(pb.reserved)
 }
+
+func (a *Accounting) getPeersBalances() (map[string]int64, error) {
+	peersBalances := make(map[string]int64)
+
+	for peer, balance := range a.balances {
+		peersBalances[peer] = balance.balance
+	}
+
+	err := a.store.Iterate("balance_", func(key, val []byte) (stop bool, err error) {
```

Comment:
Should we perhaps extract `balance_` to a constant? it appears now in at least 3 places.